client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js

```diff
@@ -3,46 +3,109 @@
  */
 import React from 'react';
 import styled from '@emotion/styled';
-import { useLineItems, useTotal, renderDisplayValueMarkdown } from '@automattic/composite-checkout';
+import {
+	CheckoutCheckIcon,
+	renderDisplayValueMarkdown,
+	useLineItemsOfType,
+	useTotal,
+} from '@automattic/composite-checkout';
 import { useTranslate } from 'i18n-calypso';
 
-/**
- * Internal dependencies
- */
-import { isLineItemADomain } from '../hooks/has-domains';
-
-export default function WPCheckoutOrderSummary( { siteUrl } ) {
-	const [ items ] = useLineItems();
-
-	const firstDomainItem = items.find( isLineItemADomain );
-	const domainUrl = firstDomainItem ? firstDomainItem.sublabel : siteUrl;
-
-	return <React.Fragment>{ domainUrl && <DomainURL>{ domainUrl }</DomainURL> }</React.Fragment>;
-}
-
-export function WPCheckoutOrderSummaryTitle() {
+export default function WPCheckoutOrderSummary() {
 	const translate = useTranslate();
+	const taxes = useLineItemsOfType( 'tax' );
 	const total = useTotal();
+
 	return (
-		<CheckoutSummaryTitle>
-			<span>{ translate( 'You are all set to check out' ) }</span>
-			<CheckoutSummaryTotal>
-				{ renderDisplayValueMarkdown( total.amount.displayValue ) }
-			</CheckoutSummaryTotal>
-		</CheckoutSummaryTitle>
+		<>
+			<CheckoutSummaryTitle>{ translate( 'Purchase Details' ) }</CheckoutSummaryTitle>
+			<CheckoutSummaryFeatures>
+				<CheckoutSummaryFeaturesTitle>
+					{ translate( 'Included with your purchase' ) }
+				</CheckoutSummaryFeaturesTitle>
+				<CheckoutSummaryFeaturesList>
+					<CheckoutSummaryFeaturesListItem>
+						<WPCheckoutCheckIcon />
+						{ translate( 'Live chat and email support' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Email and live chat support' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/checkout-two-column-layout)